src/components/Card.tsx

```diff
@@ -91,9 +95,42 @@ const ActionIcon = (props: { name: string}) => {
     return <Icon name={props.name} size={iconSize} color={Colors.DARK_GRAY} />;
 };
 
+const isPostShareable = (post: Post, author: Author): boolean => {
```

Comment:
shouldn't this be a prop and calculated in the container?

Reply:
Yes, that's a good idea